test/helpers/test-bin.js

```diff
@@ -56,6 +56,16 @@ function normalizeStderr(stderr, options = {}) {
 
   const networkIPv6 = internalIp.v6.sync();
 
+  // normalize node warnings
+  normalizedStderr = normalizedStderr.replace(
+    "(node:3658) [DEP0111] DeprecationWarning: Access to process.binding('http_parser') is deprecated.",
```

Comment:
We should look how output the warning and open an issue

Reply:
hm let me see

Reply:
issue - https://github.com/spdy-http2/node-spdy/issues/380

Reply:
we can't migrate on `http2` because express doesn't support it on Node > 10, lets serialize warnings for now

Reply:
hm, it can be problem... okay let fix it here

Reply:
serialize warnings or some other fix?

Reply:
any solutions, just do it green :smile: